tuf/client/updater.py

```diff
@@ -1495,14 +1496,21 @@ def _get_metadata_file(self, metadata_role, remote_filename,
           metadata_spec_version = metadata_signable['signed']['spec_version']
           metadata_spec_major_version = int(metadata_spec_version.split('.')[0])
           code_spec_major_version = int(tuf.SPECIFICATION_VERSION.split('.')[0])
-
+          
```

Comment:
Please remove trailing whitespace

Reply:
OT: We might want to activate a corresponding lint policy (see [in-toto's pylintrc](https://github.com/in-toto/in-toto/blob/develop/pylintrc#L59)).